modules/azurerm_vm/variables.tf

```diff
@@ -65,16 +70,31 @@ variable data_disk_caching {
   default = "ReadWrite"
 }
 
+variable data_disk_storage_account_type {
+  default = "Standard_LRS"
+  description = "The type of storage to use for the managed disk. Possible values are Standard_LRS, Premium_LRS, StandardSSD_LRS or UltraSSD_LRS."
+}
+
+variable "data_disk_zones" {
+  description = "A collection containing the availability zone to allocate the Managed Disk in."
+  default     = []
+}
+
 variable os_disk_size {
   default = 64
 }
 
+variable os_disk_storage_account_type {
+  default = "Standard_LRS"
+  description = "The Type of Storage Account which should back this the Internal OS Disk. Possible values are Standard_LRS, StandardSSD_LRS and Premium_LRS. Changing this forces a new resource to be created"
+}
+
 variable os_disk_caching {
   default = "ReadOnly"
 }
 
 variable enable_accelerated_networking {
-  default = true
+  default = false
```

Comment:
Why would this default to false here? The thought would be the customer would want this by default and if they did not need it they would turn it off.

Reply:
For jump vm types it had to be set false and for nfs it's handled here - https://github.com/sassoftware/viya4-iac-azure/blob/vm-disk-config/modules/azurerm_vm/main.tf#L19